addons/uom/data/uom_data.xml

```diff
@@ -1,205 +1,103 @@
-        <field name="category_id" ref="product_uom_categ_kgm"/>
         <field name="name">g</field>
-        <field name="factor" eval="1000"/>
-        <field name="uom_type">smaller</field>
+        <field name="relative_factor" eval="1"/>
+    </record>
+    <record id="product_uom_kgm" model="uom.uom">
+        <field name="name">kg</field>
+        <field name="relative_factor" eval="1000"/>
+        <field name="relative_uom_id" ref="uom.product_uom_gram"/>
     </record>
     <record id="product_uom_ton" model="uom.uom">
-        <field name="category_id" ref="product_uom_categ_kgm"/>
-        <!-- 'tonne' is the most common spelling in english-speaking countries,
-                the alternative is 'metric ton' in the US, abbreviated as 'mt' -->
-        <field name="name">t</field>
-        <field name="factor_inv" eval="1000"/>
-        <field name="uom_type">bigger</field>
+        <field name="name">Ton</field>
+        <field name="relative_factor" eval="1000"/>
+        <field name="relative_uom_id" ref="uom.product_uom_kgm"/>
     </record>
 
     <!--Americanization of units of measure-->
-    <!-- WEIGHT -->
-    <record id="product_uom_lb" model="uom.uom">
-        <field name="name">lb</field>
-        <field name="category_id" ref="product_uom_categ_kgm"/>
-        <field name="factor">2.20462</field>
-        <field name="uom_type">smaller</field>
-    </record>
-    <record id="product_uom_oz" model="uom.uom">
-        <field name="name">oz</field>
-        <field name="category_id" ref="product_uom_categ_kgm"/>
-        <field name="factor">35.274</field>
-        <field name="uom_type">smaller</field>
-    </record>
-
     <!-- LENGTH -->
     <record id="product_uom_inch" model="uom.uom">
-        <field name="name">in</field>
-        <field name="category_id" ref="uom_categ_length"/>
-        <field name="factor">39.3701</field>
-        <field name="uom_type">smaller</field>
+        <field name="name">inch</field>
+        <field name="relative_uom_id" ref="uom.product_uom_meter"/>
+        <field name="relative_factor" eval="0.0254"/>
+        <field name="active" eval="False"/>
     </record>
     <record id="product_uom_foot" model="uom.uom">
-        <field name="name">ft</field>
-        <field name="category_id" ref="uom_categ_length"/>
-        <field name="factor">3.28084</field>
-        <field name="uom_type">smaller</field>
-    </record>
-    <record id="product_uom_yard" model="uom.uom">
-        <field name="name">yd</field>
-        <field name="category_id" ref="uom_categ_length"/>
-        <field name="factor">1.09361</field>
-        <field name="uom_type">smaller</field>
-    </record>
-    <record id="product_uom_mile" model="uom.uom">
-        <field name="name">mi</field>
-        <field name="category_id" ref="uom_categ_length"/>
-        <field name="factor_inv" eval="1609.34"/>
-        <field name="uom_type">bigger</field>
+        <field name="name">foot</field>
```

Comment:
All other uoms are always named in a plural form, so why does these one are singular instead?
I wonder if keeping them as `in`/`ft` wouldn't be better as it doesn't bother with singular/plural.